lib/wasi-types/src/types.rs

```diff
@@ -233,6 +233,7 @@ pub mod net {
     #[repr(C)]
     pub struct __wasi_addr_t {
         pub tag: Addressfamily,
+        pub _padding: u8,
```

Comment:
Please add a comment to the `_padding` field explaining why it is there.

Reply:
@theduke if we are manually adding padding bytes, should we mark the struct as `#[repr(packed)]` and have an `assert_eq!(std::mem::size_of::<__wasi_addr_t>(), 42)` test?